pkgs/development/tools/libsigrokdecode/default.nix

```diff
@@ -9,7 +9,11 @@ stdenv.mkDerivation rec {
     sha256 = "1h1zi1kpsgf6j2z8j8hjpv1q7n49i3fhqjn8i178rka3cym18265";
   };
 
-  nativeBuildInputs = [ pkg-config ];
+  postPatch = lib.optionalString (python3.pythonAtLeast "3.8") ''
```

Comment:
Why conditional? We don't fancy conditional patching.

Reply:
no special reason, I'll remove the conditional and put a comment